Cargo.toml

```diff
@@ -54,7 +54,7 @@ lazy_static = "1"
 peeking_take_while = "0.1.2"
 quote = { version = "1", default-features = false }
 regex = "1.0"
-which = ">=1.0, <3.0"
+which = { version = ">=1.0, <3.0", optional = true }
```

Comment:
What about just leaving this change, then the feature will be automatically called `which` (you get a feature for each optional dependency).